src/applications/mhv-secure-messaging/containers/EditContactList.jsx

```diff
@@ -152,6 +157,30 @@ const EditContactList = () => {
     [isMinimumSelected],
   );
 
+  const goBackButton = () => {

```

Comment:
I would suggest declaring this as a component, not a function. 
```suggestion
  const GoBackButton = () => {

```